.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -197,27 +203,29 @@ steps:
       --features "${{ parameters.FEATURES }}" \
       --scripts-exclude "${{ parameters.SCRIPTS_EXCLUDE }}" \
       --features-exclude "${{ parameters.FEATURES_EXCLUDE }}" \
-      --specific-param='${{ parameters.SPECIFIC_PARAM }}' \
-      --affinity='${{ parameters.AFFINITY }}' \
+      --specific-param="${{ parameters.SPECIFIC_PARAM }}" \
+      --affinity="${{ parameters.AFFINITY }}" \
       --build-reason ${{ parameters.BUILD_REASON }} \
       --repo-name ${{ parameters.REPO_NAME }} \
       --mgmt-branch ${{ parameters.MGMT_BRANCH }} \
       --stop-on-failure ${{ parameters.STOP_ON_FAILURE }} \
       --retry-times ${{ parameters.RETRY_TIMES }} \
       --dump-kvm-if-fail ${{ parameters.DUMP_KVM_IF_FAIL }} \
       --requester "${{ parameters.REQUESTER }}" \
-      --max-execute-seconds $((${{ parameters.MAX_RUN_TEST_MINUTES }} * 60))
-
-      TEST_PLAN_ID=`cat new_test_plan_id.txt`
-
-      echo "Created test plan $TEST_PLAN_ID"
+      --max-execute-seconds $((${{ parameters.MAX_RUN_TEST_MINUTES }} * 60)) \
+      '
 
-      echo -e "\033[33mSONiC PR system-level test is powered by SONiC Elastictest, for any issue, please send email to sonicelastictest@microsoft.com \033[0m"
-      echo -e -n "\033[33mPlease visit Elastictest page \033[0m"
-      echo -n "$(FRONTEND_URL)/scheduler/testplan/$TEST_PLAN_ID "
-      echo -e "\033[33mfor detailed test plan progress \033[0m"
+      TEST_PLAN_ID_LIST=( $(cat new_test_plan_id.txt) )
+      for TEST_PLAN_ID in "${TEST_PLAN_ID_LIST[@]}"
+      do
+          echo "Created test plan $TEST_PLAN_ID"
+          echo -e "\033[33mSONiC PR system-level test is powered by SONiC Elastictest, for any issue, please send email to sonicelastictest@microsoft.com \033[0m"
+          echo -e -n "\033[33mPlease visit Elastictest page \033[0m"
+          echo -n "$(FRONTEND_URL)/scheduler/testplan/$TEST_PLAN_ID "
+          echo -e "\033[33mfor detailed test plan progress \033[0m"
+          echo "##vso[task.setvariable variable=TEST_PLAN_ID]$TEST_PLAN_ID"
+      done
 
```

Comment:
What's the output if create multiple test plans. If it was not easy to watch, can we group each test plan creation message?

Reply:
agree, SONiC PR system-level msg can only print once